.eslintrc.json

```diff
@@ -69,9 +69,9 @@
         "modules/**"
       ],
       "rules": {
-        "no-jquery/no-other-methods": "error",
+        "no-jquery/no-other-methods": "warn",
```

Comment:
why are these being downgraded from error to warn?

Reply:
Some jquery plugins are still required (Bootstrap tooltip) and Dynamic Table. If you remove tgem in the future then those can be converted back  to error.